packages/block-editor/src/hooks/style.js

```diff
@@ -369,6 +388,12 @@ const elementTypes = [
 ];
 
 function useBlockProps( { name, style } ) {
+	const { themeDirURI } = useSelect( ( select ) => {
```

Comment:
Not sure about this either.

For the record, this PR should just take care of theme.json/global styles but it would be good to know in advance how we'll deal with blocks.

These blocks could come from template or pattern files as well.